### PR TITLE
CP-956 Add badges to README. w_flux

### LINKED ISSUE #29: Pub badge

### PR DESCRIPTION
**Pull request by maxwellpeterson-wf**
## Issue
Fixes #29 

## Changes
**Docs:**
- Add pub, travis, codecov badges to README

## Areas of Regression
- None

## Testing
- Docs only change

@trentgrover-wf 
@evanweible-wf 
@dustinlessard-wf 
FYI @jayudey-wf

### CODE REVIEW

**Comment by evanweible-wf** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ w_flux
 
 > A Dart app architecture library with uni-directional data flow inspired by [RefluxJS](https://github.com/reflux/refluxjs) and Facebook's [Flux](https://facebook.github.io/flux/).
 
+[![Pub](https://img.shields.io/pub/v/w_flux.svg)](https://pub.dartlang.org/packages/w_flux) [![Build Status](https://travis-ci.org/Workiva/w_flux.svg?branch=master)](https://travis-ci.org/Workiva/w_flux) [![codecov.io](http://codecov.io/github/Workiva/w_flux/coverage.svg?branch=master)](http://codecov.io/github/Workiva/w_flux?branch=master)
```
Comment (trentgrover-wf) [inline README.md]: this ordering isn't consistent with other repos (badges go before description):
https://github.com/Workiva/fluri

Comment (maxwellpeterson-wf) [inline README.md]: Oops. Fixed

**Comment by evanweible-wf** [comment]
+1

**Comment by trentgrover-wf** [comment]
+1
@jayudey-wf docs only change

**Comment by jayudey-wf** [comment]
QA +1 doc change only

Merging into master

**Pull request merged**